nexus/tests/integration_tests/schema.rs

```diff
@@ -218,31 +269,53 @@ impl<'a> tokio_postgres::types::FromSql<'a> for AnySqlType {
         if f32::accepts(ty) {
             return Ok(AnySqlType::Float4(f32::from_sql(ty, raw)?));
         }
+        if serde_json::value::Value::accepts(ty) {
```

Comment:
Urgh. So I feel like I'm basically implementing OID checking here, which I'd rather do via a match statement? But reading https://docs.rs/tokio-postgres/0.7.10/tokio_postgres/types/index.html , I'm struggling to find a way to match on the [Type](https://docs.rs/tokio-postgres/0.7.10/tokio_postgres/types/struct.Type.html) type, rather than have this chain of if-statements.

- [Type::kind](https://docs.rs/tokio-postgres/0.7.10/tokio_postgres/types/enum.Kind.html) returns an enum of possible "flavors of type", but this isn't enough to infer the conversion to one of our "rust types" in the `AnySqlType` variant.
- [Type::oid](https://docs.rs/tokio-postgres/0.7.10/tokio_postgres/types/struct.Type.html#method.oid) can return a `u32` that exactly tells us the resulting oid. But... it's not a const method? So I can't actually access it from all the constant types declared in the "Associated Constants" of https://docs.rs/tokio-postgres/0.7.10/tokio_postgres/types/struct.Type.html#implementations. 

Frustratingly, this info **does** exist in the postgres_types crate: https://docs.rs/postgres-types/0.2.6/src/postgres_types/type_gen.rs.html#207 but it isn't exposed publically.